### DISCUSSION TITLE
Preset for Turbo model

### DISCUSSION THREAD

**Discussion by E2GO** [mashb1t/Fooocus#33] [Q&A] [answered]
What preset better to use with Turbo models?

**Comment by mashb1t** [comment, marked as answer]
Hey @E2GO, sorry just saw your comment now. To be honest either the official [turbo](https://github.com/mashb1t/Fooocus/blob/v2.4.0/presets/turbo.json) one but probably none of them, you'll most likely have to create your own and set override steps to the amount of steps the Turbo model has been trained on / is optimized for.
